test/fastly/client_test.rb

```diff
@@ -29,6 +29,15 @@
       assert_equal "Invalid auth credentials. Check username/password.", e.message
     end
 
+    it 'surfaces a deprecation message when a username or password is provided' do
+      stub_request(:any, /api.fastly.com/).
+        to_return(body: JSON.generate(i: "dont care"), status: 200)
```

Comment:
I don't think these lines are necessary anymore. The tests should pass without the stubbing.

Reply:
The stub is required by Webmock